core/locale/src/commonMain/kotlin/com/adammcneilly/pocketleague/core/locale/CommonLocale.kt

```diff
@@ -0,0 +1,7 @@
+package com.adammcneilly.pocketleague.core.locale
+
+/**
+ * Given a [countryCode], convert it to a relevant display name. For example,
+ * "us" should return "United States".
+ */
+expect fun getCountryDisplayName(countryCode: String): String
```

Comment:
I think we should make a `LocaleHelper` interface instead, and the expect fun can provide the instance of that interface.